src/common-components/tests/EmbeddedRegistrationRoute.test.jsx

```diff
@@ -47,10 +47,13 @@ describe('EmbeddedRegistrationRoute', () => {
     let embeddedRegistrationPage = null;
 
     await act(async () => {
-      embeddedRegistrationPage = await mount(routerWrapper());
+      const { container } = await render(routerWrapper());
+      embeddedRegistrationPage = container;
```

Comment:
Would this work? if it would, please change in all places
```suggestion
      const { container: embeddedRegistrationPage } = await render(routerWrapper());
```

Reply:
It won't work.